woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1370,7 +1454,7 @@ def test_accessor_set_index(sample_df):
     assert sample_df.ww.index is None
     if isinstance(sample_df, pd.DataFrame):
         # underlying index not set for Dask/Koalas
-        assert (sample_df.index == range(4)).all()
+        assert (sample_df.index == sample_df['full_name']).all()
```

Comment:
Maybe add a comment here that we are checking to make sure that the underlying index doesn't change when setting the index to None? Might not be clear why we are testing this a few months in the future.

Reply:
Added comment has an extra `i` in it. `Woodworki` -> `Woodwork`